docs/reference/release-notes/release-notes.md

```diff
@@ -6,23 +6,21 @@ description: "Release notes for Camunda 8 and its components."
 
 Camunda 8 release notes include notable new and improved features, enhancements, and bug fixes. Release notes are separated by minor release pages and include alphas released during the development cycle.
 
-[8.5.0](/reference/release-notes/850.md)
+[8.6](/reference/release-notes/860.md)
+[8.5](/reference/release-notes/850.md)
 
 ## Changelogs
 
-Changelogs for Camunda 8, including alphas, are also available on [GitHub](https://github.com/camunda/camunda-platform/releases). This includes release assets for Zeebe, Operate, Tasklist, and Identity.
-
-The current changelog can be found on the [Camunda repository](https://github.com/camunda/camunda-platform/releases/latest) and [Camunda Helm repository](https://github.com/camunda/camunda-platform-helm/releases/latest).
-
-[Update guides](/self-managed/operational-guides/update-guide/introduction.md) include links to both changelogs and release blogs.
+Changelogs for Camunda 8, including alphas, are available on [GitHub](https://github.com/camunda/zeebe/releases) and [Camunda Helm repository](https://github.com/camunda/camunda-platform-helm/releases).
 
 Most releases and changelogs are available through the [Camunda Platform repo](https://github.com/camunda/camunda-platform), however, the following component release notes are available as linked below:
```

Comment:
We have a confusing split right now between what is available on the camunda-platform repo and what is on the Zeebe repo.